.gitignore

```diff
@@ -44,3 +44,8 @@ coverage/*
 ###< symfony/web-server-bundle ###
 
 .env
+
+# Ignore mounts
+var
+installer
+web
```

Comment:
```suggestion
web
upgrades
```
Not needed?

Reply:
That's... odd.  It's there in the template-builder. I'll rerun the build and see if that changes anything.